.eslintignore

```diff
@@ -29,6 +29,7 @@
 /tmp
 .next
 build
+apps/*/dist
```

Comment:
maybe we can make this a general rule, deduplicate with the one above

```suggestion
dist
```

Reply:
Oh ok, now I see, I thought you meant to align it with `build` (the one above) 😄